data.json

```
@@ -178,5 +178,11 @@
         "githubUsername": "aniketsingh98571",
         "imageUrl": "https://user-images.githubusercontent.com/63097956/93450123-351c2600-f8f3-11ea-8964-bd44fd12dc0f.png",
         "issueNumber": 88
-    }
+    },
+    {
+    "name": "Nirmal Silwal",
+    "githubUsername": "NirmalSilwal"
```

Missing comma, should be

```suggestion
    "githubUsername": "NirmalSilwal",
```

---

My bad! thanks for pointing it out